event_model/__init__.py

```diff
@@ -2402,6 +2402,7 @@ def __call__(
         time=None,
         uid=None,
         validate=True,
+        **kwargs,
```

Comment:
I think we should handle this the same way as RunStart (without the unpacking)

https://github.com/bluesky/event-model/blob/ef8ee4b6040e647883d8adbe6c5fa9da33ac984a/event_model/__init__.py#L2512

I'm happy to change it to unpacking if others prefer, but I think they should be consistent